community/php7/php7-module.conf

```diff
@@ -1,4 +1,4 @@
-LoadModule php7_module modules/mod_php7.so
+LoadModule php7_module modules/libphp7.so
```

Comment:
in php5 it named libphp5. So I decided not to change it's name and stay it as is.